ballerina/tests/client_endpoint_test.bal

```diff
@@ -551,7 +552,7 @@ public function testDeleteFile() returns error? {
             test:assertFail(msg = "Error while closing stream in `get` operation.");
         }
     } else {
-       test:assertFail(msg = "Found unexpected output type ");
+       test:assertTrue(true);
```

Comment:
you can remove this condition because the outcome of that assert is obvious right?

Reply:
Replaced with error message validation.